### PR TITLE
Feature specs

### PR DESCRIPTION
**Pull request by elreyatee**

### CODE REVIEW

File: spec/models/video_spec.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
   it { should have_many(:reviews).order("created_at DESC") }
 
   describe "search_by_title" do
```
Comment (tomtomecek) [inline spec/models/video_spec.rb]: This describe block's message should be slightly different - Read more [here](http://www.betterspecs.org/#describe).

File: spec/support/macros.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+def sign_in(some_user=nil)
```
Comment (tomtomecek) [inline spec/support/macros.rb]: You could use your knowledge of memoization:

``` ruby
def sign_in(user = nil)
  user ||= Fabricate(:user)
  visit sign_in_path
  fill_in "Email", with: user.email
  fill_in "Password", with: user.password
  click_button "Sign in"
end
```

Comment (elreyatee) [inline spec/support/macros.rb]: You're right, changed code to use this technique

File: spec/models/video_spec.rb (hunk 1 above)
Comment (elreyatee) [inline spec/models/video_spec.rb]: Good advice, changed to:

describe "#search_by_title" do

Comment (tomtomecek) [inline spec/models/video_spec.rb]: Actually, `describe ".search_by_title"` because its a class method.

**Pull request merged**